deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -0,0 +1,180 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Custom least privilege IAM policy for linking environments to dataall
+Parameters:
+  AwsAccountId:
+    Description: AWS AccountId of the dataall INFRASTRUCTURE account that we wish to link this environment with.
+    Type: String
+  PolicyName:
+    Description: IAM policy name (copy/paste from the UI)
+    Type: String
+    Default: 'DataAllCustomCDKPolicy'
+  EnvironmentResourcePrefix:
```

Comment:
We are barely using this prefix in our policies and we could really restrict access to prefixed resources only (e.g. all S3 buckets created will be repfixed with this resource)

Reply:
For future enhancements